src/file_view_pool.cpp

```diff
@@ -172,7 +172,7 @@ namespace libtorrent { namespace aux {
 		auto& lru_view = m_files.get<1>();
 		if (lru_view.size() == 0) return {};
 
-		auto mapping = std::move(lru_view.back().mapping);
```

Comment:
It would be better to fix this to *actually* move the mapping. I imagine `lru_view.back()` returns an immutable reference.